core/src/banking_stage/decision_maker.rs

```diff
@@ -138,7 +138,9 @@ impl DecisionMaker {
 
 impl BankingStageMonitor for DecisionMaker {
     fn status(&mut self) -> BankingStageStatus {
-        if matches!(
+        if self.poh_recorder.read().unwrap().is_exited.load(Relaxed) {
```

Comment:
Would be good to just clone `is_exited` off of `poh_recorder`. It's already wrapped in an Arc, so we can avoid locking `poh_recorder` each time we check status.

Reply:
nice suggestion!: a9916d3003f200d6e082e9cb8166080dc9d983d8